lib/bitrise/response.rb

```diff
@@ -0,0 +1,22 @@
+module Bitrise
+  class Response
+    def initialize(json)
+      @json = json
+    end
+  end
+
+  class Paging
+    attr_reader :total_item_count, :page_item_limit
```

Comment:
`next` is also necessary?

If there are more apps than the number specified by `limit` query parameter in request, a request with second page onwards is necessary.  In that case, it should make a paging request with the slug returned by the Bitrise API as the `paging` request parameter.

https://api-docs.bitrise.io/#/application/app-list

Reply:
Yes, I missing it. I will fix it 🔥

Reply:
`next` is added at [83d0a8f](https://github.com/mataku/bitrise-client/pull/6/commits/83d0a8ff4524e2774f25dd14649242037b1b50da)